server/models/db/timeslot.js

```diff
@@ -0,0 +1,17 @@
+const _ = require('lodash');
+
+const { db } = require('../../lib/connection');
+const Tables = require('../../constants/tables.json');
+
+const timeslotObject = require('../data/Timeslot');
```

Comment:
Hi Peter,
I would recommend renaming this to `Timeslot` to differentiate it from the other types of variables and clearly illustrate that it is an object.